pkg/azuredisk/device_perf_optimization_linux.go

```diff
@@ -149,10 +149,7 @@ func echoToFile(content string, filePath string) (err error) {
 }
 
 func getOptimalDeviceSettings(nodeInfo *NodeInfo, diskSkus map[string]map[string]DiskSkuInfo, perfProfile string, accountType, diskSizeGibStr, diskIopsStr, diskBwMbpsStr string) (queueDepth, nrRequests, scheduler, maxSectorsKb, readAheadKb string, err error) {
-	klog.V(2).Infof("Calculating perf optimizations for rofile %s accountType %s diskSize", perfProfile, accountType, diskSizeGibStr)
-
-	err = nil
-
+	klog.V(12).Infof("Calculating perf optimizations for rofile %s accountType %s diskSize", perfProfile, accountType, diskSizeGibStr)
```

Comment:
V(2)?

Reply:
I was pointing out the V(12), I thought it was a typo and the highest was V(5)?

Reply:
V(12) is for debugging, no need to print every log in our driver by default.

Reply:
we still print v(5) logs by default now, set as v(6 or bigger) would not print now.